### ISSUE TITLE
Reconcile Python versions between SS & SSOS before next release

### ISSUE THREAD

**Issue by bitcoinprecept** [SeedSigner/seedsigner-os#44]
TLDR: @jean identified [conflicting Python versions](https://github.com/buildroot/buildroot/blob/b634d504cb5a0b621f115f36c50a712e85fc8027/package/python3/python3.hash) when working on review of Seedsigner [PR 347](https://github.com/SeedSigner/seedsigner/pull/347) @kdmukai says "So I'd say: IF SeedSigner OS can trivially version up to 3.10.10, then we should PR that in for the next release. But if it can't, THEN we'll make sure the manual build steps match whatever minor version SeedSigner OS is stuck at."

Additional Context:
> Jean Do:
Just a note that I've run through pr #347 today (how-to install python 3.10 on the pi board) from start to finish and didnt have any critical problems (though I did leave notes). ACK-tested from me.

I did this because during dev/testing I've been switching between virtualenvs with python 3.7 and python 3.10... and I've finally moved on to 3.10 only, which is ahead of this pr and therefore ahead of what other devs might be using.  I feel #347 is ready as soon as other devs would agree.

> Jean Do:
@klockenga , I didnt run into problems with numpy 1.21.1 and was not forced to use 1.21.6, but I recall this being mentioned in the past.

> Jean Do:
@kdmukai, your notes helped me to install python 3.10 originally (2months ago?) and for some reason, I ended up originally installing 3.10.4 instead of 3.10.10.  That caught my eye this morning and but I followed your pr exactly... but I'm wondering how we chose different versions.   How can I verify exactly what is included in the seedsigner-os 0.6.0 image that everyone up-to-date is running?

added: I'm thinking that I originally followed seedsigner-os/opt/buildroot commit b634d.... and stumbled onto this
https://github.com/buildroot/buildroot/blob/b634d504cb5a0b621f115f36c50a712e85fc8027/package/python3/python3.hash

I realize that #347 is still a draft.  If instead of 3.10.10 we should have manual instructions for 3.10.4 (because that's what all of our seedsigner-os users have installed), then just confirm and I'll restart my verification run of your #347 pr for that earlier python 3.10.

Keith says:
I can't recall if there was any specific reason for 3.10.10 vs 3.10.4, other than opting for the latest bugfixes. I think I noted somewhere that there should be a matching PR in the SeedSigner OS to version up to the matching minor release, if possible.

It would be ideal if they matched, but I'm not that worried about it as I wasn't even aware that SeedSigner OS was using 3.10 in the first place. I think my local dev build was still on 3.8 or 3.9.

So I'd say: IF SeedSigner OS can trivially version up to 3.10.10, then we should PR that in for the next release. But if it can't, THEN we'll make sure the manual build steps match whatever minor version SeedSigner OS is stuck at.

Jean says: ty, I'll remember this point when nick or dt add their input.  my raincheck to rerun this morning's end-to-end manual-installation verification of #347 for another minor version of python remains open, if it's not trivial to bump in ss-os

**Comment by DesobedienteTecnologico** [comment]
Hi, thanks for your issue.

As soon I get hands on in reproducible builds the Python version can be changed to any version needed. I just commented in this issue too:
https://github.com/SeedSigner/seedsigner/pull/347#issuecomment-1523837935

Regards
